dev/tests/integration/testsuite/Magento/Catalog/Model/ProductGettersTest.php

```diff
@@ -217,7 +217,7 @@ public function testGetAttributeTextArray()
         $expected = [
             'Option 2',
             'Option 3',
-            'Option 4 &quot;!@#$%^&amp;*'
```

Comment:
It would be a good idea to cover more cases with the test. Possibly combinations of plain/html setting for attribue